app/views/home/index.html.erb

```diff
@@ -6,35 +6,22 @@
     </div>
   </div>
   <!-- "Welcome" section -->
-  <section class="homepage-welcome-banner dm-gradient-banner-318" role="region" aria-label="Homepage welcome banner">
-    <div class="grid-container height-full">
-      <div class="grid-row height-full flex-align-center">
-        <div class="grid-col-12">
-          <h1 class="usa-prose-h1 text-white margin-top-4 margin-bottom-4 desktop:margin-top-0">
+  <section class="usa-section padding-top-3 padding-bottom-7 bg-primary-darker" role="region" aria-label="Homepage welcome banner">
```

Comment:
@camillevilla just went ahead and removed the `homepage-welcome-banner` css classes